### PR TITLE
frontend: set darkmode colour scheme for ExchangeSelectionRadio

### PR DESCRIPTION
**Pull request by shonsirsha**
hardcoded some CSS values in order to support darkmode for this component
(ExchangeSelectionRadio).

Used light icons for the Bank and CreditCard icons.

Preview:

Screenshot 2023-03-20 at 12 54 37

### CODE REVIEW

File: frontends/web/src/routes/buy/components/exchangeselectionradio.module.css [hunk 1]
```diff
@@ -129,7 +129,7 @@
 /* checked */
 
 .radio[aria-checked="true"] {
-    background-color: var(--color-light-skyblue);
+    background-color: var(--background);
```
Comment (thisconnect) [inline frontends/web/src/routes/buy/components/exchangeselectionradio.module.css:132]: in light mode `--background` is using `--color-lightgray`, but here it was `--color-light-skyblue` 🤔 

is this for focusing the background of the whole option?
do we need a new CSS variable `--background-focus` or similar?

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/routes/buy/components/exchangeselectionradio.module.css (hunk 1 above)
Comment (shonsirsha) [reply frontends/web/src/routes/buy/components/exchangeselectionradio.module.css:132]: > is this for focusing the background of the whole option?
Yes exactly.

> do we need a new CSS variable --background-focus or similar?
That'd be good, I think. As lightmode uses skyblue and in darkmode it obviously won't. WDYT?

Comment (thisconnect) [reply frontends/web/src/routes/buy/components/exchangeselectionradio.module.css:132]: makes sense to me

**Comment by shonsirsha** [comment]
Rebased and merged conflicts solved :)

**Review by thisconnect — APPROVED** [review]
untested LGTM

File: frontends/web/src/routes/buy/components/exchangeselectionradio.module.css (hunk 1 above)
Comment (jadzeidan) [reply frontends/web/src/routes/buy/components/exchangeselectionradio.module.css]: I was testing it out and I think it is more intuitive for the focus background to be the lighter one. So like this:

image

Comment (shonsirsha) [reply frontends/web/src/routes/buy/components/exchangeselectionradio.module.css]: agree, looks better too @jadzeidan . Thanks.

**Pull request merged**